### PR TITLE
B0.23

### PR DESCRIPTION
**Pull request by maaktweluit**

### CODE REVIEW

**Comment by maaktweluit** [comment]
Closing, needs a merge commit so made a new branch

File: scripts/task_api_tests/task-api-readme.md [hunk 1]
```diff
@@ -56,7 +56,7 @@ The golem block of the JSON is meant for the input Golem needs, these are the sa
 
 App id is the unique identifier of the app including its version.
 You can get the build in app_id's from the logs when starting golem
-`daec55c08c9de7b71bf4ec3eb759c83b` is `golemfactory/blenderapp:0.7.2` the only available option at the time.
+`6b39331ac73484ff596447cefdfba9e5` is `golemfactory/blenderapp:0.7.2` the only available option at the time.
```
Comment (Wiezzel) [inline scripts/task_api_tests/task-api-readme.md:59]:
```suggestion
`6b39331ac73484ff596447cefdfba9e5` is `golemfactory/blenderapp:0.7.3` the only available option at the time.
```
?

**Review by Wiezzel — APPROVED** [review]
OK

File: scripts/task_api_tests/task-api-readme.md (hunk 1 above)
Comment (maaktweluit) [reply scripts/task_api_tests/task-api-readme.md:59]: Good one! will make a PR to develop for this, not blocking release

Comment (maaktweluit) [reply scripts/task_api_tests/task-api-readme.md:59]: updated the b0.23-to-develop PR

**Comment by jiivan** [comment]
> Closing, needs a merge commit so made a new branch

Actually you don't have to close it. It will do automatically once everything is merged (even if in different PR)

**Comment by maaktweluit** [comment]
Good to know, thanks!

**Pull request closed without merging**